rcl/test/cmake/rcl_add_custom_gtest.cmake

```diff
@@ -43,13 +43,16 @@ set(rcl_add_custom_gtest_INCLUDED TRUE)
 # :type LIBRARIES: list of strings
 # :param AMENT_DEPENDENCIES: list of depends to pass ament_target_dependencies
 # :type AMENT_DEPENDENCIES: list of strings
-#
+# :param WORKING_DIRECTORY: the working directory for invoking the
+#   executable in, default defined by ``ament_add_test()``
+# :type WORKING_DIRECTORY: string
+
 # @public
 #
 macro(rcl_add_custom_gtest target)
   cmake_parse_arguments(_ARG
     "SKIP_TEST;TRACE"
-    "TIMEOUT"
+    "TIMEOUT;WORKING_DIRECTORY"
```

Comment:
It doesn't look like this new parameter is being used anywhere. Do we really need this change?

Reply:
A leftover from a previous attempt. I rolled back all the changes to this file in https://github.com/ros2/rcl/pull/508/commits/c1f4c3ce1dc69cf0d29cf1c72873d582efa7a557